features/config-has.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Check whether the wp-config.php file has a certain constant or variable
+
+  Background:
+    Given a WP install
+
+  Scenario: Check the existence of an existing wp-config.php constant or variable
+    When I run `wp config has DB_NAME`
+    Then STDOUT should be empty
+    And the return code should be 0
```

Comment:
I missed this but don't need this "return code" either on `run` but let's leave it for now!

Reply:
As this test is about testing the return code, I think it's preferable to be explicit here.